components/TabList/tab-list.tsx

```diff
@@ -37,13 +37,14 @@ const TabList = ({ tabList, pageId, selectedTab }: TabListProps) => {
           role="tab"
           aria-selected={selectedTab === tab.name.toLowerCase() ? "true" : "false"}
           data-state={selectedTab === tab.name.toLowerCase() ? "active" : "inactive"}
-          tabIndex={-1}
+          tabIndex={0} // Added tabIndex for accessibility
           key={index}
-          className={`tool-list-item border-b-2 transition-all ease-in-out ${
+          className={clsx(
+            "tool-list-item border-b-2 transition-all ease-in-out",
```

Comment:
no need to refactor this as it's not related to the issue. Feel free to open a separate issue for this.

Reply:
It was automatically formatted when i ran `npm run push`